packages/@okta/vuepress-site/docs/reference/architecture-center/mmod/lab-2-create-resources/index.md

```diff
@@ -0,0 +1,197 @@
+
+1. Append the following to `groups.tf` in GitHub, then commit your changes.
+
+   ```hcl
+   resource "okta_group" "test_group" {
+     name        = "Test Group"
+     description = "For testing and demonstration"
+   }
+   ```
+
+2. In your workspace **Overview** page, observe that a new plan runs, changes are applied automatically, and **Resources changed** shows `+1 ~0 -0` to reflect the added resource. The **Latest Run** status is Applied.
+
+3. Append the following to `grouprules.tf` in GitHub, then commit your changes.
+
+   ```hcl
+   resource "okta_group_rule" "test_rule" {
+     name              = "Test Rule"
+     status            = "ACTIVE"
+     group_assignments = ["${okta_group.test_group.id}"]
+     expression_type   = "urn:okta:expression:1.0"
+     expression_value  = "String.startsWith(user.firstName,\"somename\")"
+   }
+   ```
+
+4. In your workspace **Overview** page, observe that a new plan runs, changes are applied automatically, and **Resources changed** shows `+1 ~0 -0` to reflect the added resource. The **Latest Run** status is Applied.
+
+## Add a new customer application portal resource
+
+Okta is a customizable, secure, drop-in solution to add authentication and authorization services to your applications. You can connect any application on any device and define how you want your users to sign in. A customer portal is an interface through which your customers access your application.
+
+Use Customer portal resources to create and configure different portals. They are defined in `modules/applications/customer-portal.tf`.
+
+In this exercise, you add a new [OIDC application portal](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/app_oauth) and verify the results in your workspace Overview page.
+
+1. Append the following to `modules/applications/customer-portal.tf` in GitHub, then commit your changes.
+
+   ```hcl
+   resource "okta_app_oauth" "example" {
+     status         = "ACTIVE"
+     label          = "Example Portal"
+     type           = "web"
+     redirect_uris  = ["https://localhost:44378/signin-oidc", "https://localhost:44378/Customer", "https://oauth.pstmn.io/v1/callback"]
+     post_logout_redirect_uris = ["https://localhost:44378/signout-callback-oidc"]
+     login_uri      = "https://localhost:4437811/signin-oidc"
+     grant_types    = ["client_credentials", "authorization_code", "refresh_token"]
+     response_types = ["code"]
+     consent_method = "TRUSTED"
+     refresh_token_rotation = "ROTATE"
+     skip_users = true
+     skip_groups = true
+   }
+   ```
+
+2. In your workspace **Overview** page, observe that a new plan runs, changes are applied automatically, and **Resources changed** shows `+1 ~0 -0` to reflect the added resource. The **Latest Run** status is Applied.
+
+## Add a new application sign-on policy resource
```

Comment:
they are called authentication policies now - not application sign-on policies unless you are referencing Classic Engine.

```suggestion
## Add a new authentication policy resource
```